docs/reference/commandline/run.md

```diff
@@ -262,6 +262,13 @@ https://docs.docker.com/engine/installation/binaries/#/get-the-linux-binary)),
 you give the container the full access to create and manipulate the host's
 Docker daemon.
 
+On Windows, the paths must be specified using Windows-style semantics. Windows
+containers do not support the target of a volume or bind-mount being pre-
+populated in the container image.
```

Comment:
It c:\target exists in the image, you will get back an error such as the following (the directory is not empty bit specifically)

```
PS C:\> docker run -v "c:\foo:c:\Program Files" microsoft/nanoserver cmd /s /c type "c:\Program Files\somefile.txt"
e:\go\src\github.com\docker\docker\bundles\docker.exe: Error response from daemon: container c8ac5e4ece96027744088b4a19694e5a9e1596bd9f5f16055018e2d38f8d215b encountered an error during CreateContainer: failure in a Windows system call: The directory is not empty. (0x91) extra info: {"SystemType":"Container","Name":"c8ac5e4ece96027744088b4a19694e5a9e1596bd9f5f16055018e2d38f8d215b","Owner":"docker","IsDummy":false,"VolumePath":"\\\\?\\Volume{a12acb49-a5f4-11e6-a9e3-00155d768276}","IgnoreFlushesDuringBoot":true,"LayerFolderPath":"C:\\control\\windowsfilter\\c8ac5e4ece96027744088b4a19694e5a9e1596bd9f5f16055018e2d38f8d215b","Layers":[{"ID":"fefb26f1-cf06-5de2-979b-54e24b4df8e7","Path":"C:\\control\\windowsfilter\\3ad1811989c1c273f1216de7ff6e36cc4374e031bead3fb32aaddcb19386ac6b"}],"HostName":"c8ac5e4ece96","MappedDirectories":[{"HostPath":"c:\\foo","ContainerPath":"c:\\program files","ReadOnly":false,"BandwidthMaximum":0,"IOPSMaximum":0}],"HvPartition":false,"EndpointList":["55eedba6-5d07-47f5-b1ee-bc09e4609629"],"Servicing":false,"AllowUnqualifiedDNSQuery":true}.
PS C:\>
```